ironfish/src/consensus/verifier.ts

```diff
@@ -446,26 +490,28 @@ export enum VerificationResultReason {
   DOUBLE_SPEND = 'Double spend',
   DUPLICATE = 'Duplicate',
   ERROR = 'Error',
-  GRAFFITI = 'Graffiti field is not 32 bytes in length',
   GOSSIPED_GENESIS_BLOCK = 'Peer gossiped its genesis block',
+  GRAFFITI = 'Graffiti field is not 32 bytes in length',
   HASH_NOT_MEET_TARGET = 'Hash does not meet target',
+  INVALID_ASSET_NAME = 'Asset name is blank',
+  INVALID_BURN = 'Attempting to burn the native asset',
```

Comment:
This is a nit so feel free to ignore, but I wonder if we should change the enum to be a bit more explicit since this error is specific to the native asset. I'm unsure about what the name should be though ¯\_(ツ)_/¯

Reply:
Good call - renamed to `NATIVE_BURN`